samples/hello-world/README.md

```diff
@@ -0,0 +1,170 @@
+# Cloud Bigtable Hello World
+
+This is a simple application that demonstrates using the Nodejs Google Cloud
+API to connect to and interact with Cloud Bigtable.
+
+**Table of Contents**
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Downloading the sample](#downloading-the-sample)
+- [Costs](#costs)
+- [Before you begin](#before-you-begin)
+  - [Installing Node dependencies](#installing-node-dependencies)
+  - [Creating a Project in the Google Cloud Platform Console](#creating-a-project-in-the-google-cloud-platform-console)
+  - [Enabling billing for your project.](#enabling-billing-for-your-project)
+  - [Enable the Cloud Bigtable APIs.](#enable-the-cloud-bigtable-apis)
+  - [Install the Google Cloud SDK.](#install-the-google-cloud-sdk)
+  - [Setting Google Application Default Credentials](#setting-google-application-default-credentials)
+- [Provisioning an instance](#provisioning-an-instance)
+- [Running the application](#running-the-application)
+- [Cleaning up](#cleaning-up)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+
+## Downloading the sample
+
+Download the sample app and navigate into the app directory:
+
+1.  Clone the [Cloud Bigtable examples repository][github-repo], to your local
```

Comment:
"Clone the repo" link and text need to change

Reply:
Done